code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -97,3 +97,17 @@
 		to_chat(user, "<span class='notice'>..and falls into view. Whew, that was a close one.</span>")
 		user.dropItemToGround(src)
 
+/obj/item/gun/ballistic/automatic/liberatorpistol
+	name = "Makeshift Pistol"
+	desc = "Harkening back to the prehistoric Liberator pistol design, this small makeshift pistol fires 10mm bullets."
```

Comment:
```suggestion
	desc = "Harkening back to before humanity achieved spaceflight on Terra, this \"Liberator\" makeshift pistol fires 10mm bullets."
```
Gives it a bit more of a spacebound-humanity-looking-back-on-earthbound-humanity feel. Also doesn't imply that we lost most of Earth's history of the mid-20th century.